api/src/main/webapp/WEB-INF/cron_base.yaml

```diff
@@ -89,6 +89,6 @@ cron:
     Check the Runtimes, Apps, and Disks of all active Workspaces and delete those created by users
     who no longer have access to the Workspaces.
   url: /v1/cron/deleteUnsharedWorkspaceEnvironments
-  schedule: every day 19:30
```

Comment:
what does this mean?

Reply:
For all cron environments except test (cron_test.yaml is what overrides this) run only once a year.  I plan to update this before the year is over.